CIP-1694/README.md

```diff
@@ -0,0 +1,1161 @@
+
+The **network group** consists of:
+* maximum block body size (`maxBBSize`)
+* maximum transaction size (`maxTxSize`)
+* maximum block header size (`maxBHSize`)
+* maximum size of a serialized asset value (`maxValSize`)
+* maximum script execution units in a single transaction (`maxTxExUnits`)
+* maximum script execution units in a single block (`maxBlockExUnits`)
+* maximum number of collateral inputs (`maxCollateralInputs`)
+
+The **economic group** consists of:
+* minimum fee coefficient (`minFeeA`)
+* minimum fee constant (`minFeeB`)
+* delegation key Lovelace deposit (`keyDeposit`)
+* pool registration Lovelace deposit (`poolDeposit`)
+* monetary expansion (`rho`)
+* treasury expansion (`tau`)
+* minimum fixed rewards cut for pools (`minPoolCost`)
+* minimum Lovelace deposit per byte of serialized UTxO (`coinsPerUTxOByte`)
+* prices of Plutus execution units (`prices`)
+
+The **technical group** consists of:
+* pool pledge influence (`a0`)
+* pool retirement maximum epoch (`eMax`)
+* desired number of pools (`nOpt`)
+* Plutus execution cost models (`costModels`)
+* proportion of collateral needed for scripts (`collateralPercentage`)
+
+The **governance group** consists of all the new protocol parameters that are introduced in this CIP:
+* governance voting thresholds ($P_1$, $P_{2a}$, $P_{2b}$, $P_3$, $P_4$, $P_{5a}$, $P_{5b}$, $P_{5c}$, $P_6$, $P_7$, $Q_1$, $Q_{2b}$, $Q_4$)
+* constitutional committee term limits
+* governance action expiration
+* governance action deposit (`govDeposit`)
+* DRep deposit amount (`drepDeposit`)
+* DRep activity period (`drepActivity`)
+
+<!-- TODO:
+  - Decide on the initial values for the new governance parameters
+
+  - Decide on coherence conditions on the voting thresholds.
+    For example, the threshold for a motion of no-confidence should arguably be higher than that of a minor treasury withdrawal.
+-->
+
+<!---------------------------    Governance Actions    ------------------------>
+
+<!---------------------------          Votes           ------------------------>
+
+### Votes
+
+Each vote transaction consists of the following:
+
+* a governance action ID
+* a role - constitutional committee member, DRep, or SPO
+* a governance credential witness for the role
+* an anchor (as defined above) for information that is relevant to the vote
+* a 'Yes'/'No'/'Abstain' vote
+
+For SPOs and DReps, the number of votes that are cast (whether 'Yes', 'No' or 'Abstain') is proportional to the Lovelace that is delegated to them at the point the
+action is checked for ratification.  For constitututional committee members, each current committee member has one vote.
+Votes from unregistered SPOs or DReps count as having zero stake.
+
+> **Warning** 'Abstain' votes are not included in the "active voting stake".
+>
+> Note that an explicit vote to abstain differs from abstaining from voting.
+> Unregistered stake that did not vote behaves like an 'Abstain' vote,
```

Comment:
At this moment, the CIP does an unsuitable job explaining the "active voting stake" and especially what is considered part of the "active voting stake" and what isn't. For example:

>  Unregistered stake that did not vote behaves like an 'Abstain' vote, while registered stake that did not vote behaves like a 'No' vote.

Do I understand correctly that if I'm delegating my ada to Alice and, for whatever reason, she cannot vote on any given governance action, her voting power will be counted towards "No" for this GA? 

The way I thought it would work is that Alice, an active DRep, her voting power is counted towards the "active voting stake," and if she doesn't vote on a GA, her vote is just not counted to any of the three "Yes," "No," "Abstain."

I hope in one of the subsequent overhauls of the CIP we could add a dedicated section for the "active voting stake."

My suggestion:

> The **"active voting stake"** is an essential concept in the Cardano blockchain's governance system. It represents the sum of all delegated voting stakes that are considered "active" and are used to determine the outcome of governance actions.
> 
> To be considered active for voting, the Lovelace contained in a transaction output must meet two criteria:
> 
> 1. It contains a registered stake credential.
> 2. The registered stake credential has delegated its voting rights to a registered DRep.
> 
> When votes are cast as 'Abstain' on a governance action, it is not considered part of the active voting stake, although it is still considered registered for incentives. 
>
> DReps must regularly vote to maintain their active status; otherwise, they will no longer be considered part of the active voting stake.

Reply:
Casting a vote as `Abstain` does not suddenly render the associated stake inactive (as the suggested prose states). The definition is really just the two points: 1) being registered, and 2) being delegated.

---

> her voting power will be counted towards "No" for this GA?

No, I would not say that it "counts as no". What we said was that it "behaves like a no".

Let me give an example. Say all the active voting stake is: Alice has 3 lovelace, Bob has 1 lovelace, Carol has 1 lovelace, Don has 1 lovelace. Let's say that we need 51% yes votes to ratify something.

Say then that: Alice votes yes, Bob vote no, Carol votes abstain, and Don doesn't vote. The amount of active voting stake is 6. But since Coral voted abstain, the denominator used for checking the threshold will be 5. The fraction of yes votes is 3/5, which is enough for ratification. Note that Bob and Don's votes did the same thing, semantically.  I would not say that Don's vote was "counted as a no", but I would say that it "behaved like a no".